Microcontroller_L432KC/README.md

```diff
@@ -0,0 +1,18 @@
+# ECE3SAT_PhaseB_EDT
+
+This code has been wrote for STM32 Nucleo L432KC.
+
+## Prerequisites
+
+You need to have an installed and configured [MbedCLI](https://os.mbed.com/docs/v5.7/tools/arm-mbed-cli.html).
+
+
+## Installation
+
+Use `mbed new .` in this directory to create a Mbed project with all configurations and dependencies in this repository.
+
+You can now use mbed command to compile these files to get the binary to flash your microcontroller
+in <project_dir>/BUILD/<target_name>/<compiler_name>/ :  `mbed compile`

```

Comment:
Finally, I would advise you to warn the user as early as possible on the fact that target-specific options are needed as detailed below.
Any random user could run each command in the order they see it without reading the surrounding text thoroughly. Thus, if you make it clear that the command `mbed compile` should be run with some options (`mbed compile -m $MC_TARGET -t $COMPILER`) then the reader will read until they understand which specific target and which compiler they should use.